guides/common/modules/con_configuring-dhcp-integration.adoc

```diff
@@ -0,0 +1,2 @@
+[id="configuring-dhcp-integration"]
+= Configuring DHCP integration
```

Comment:
To comply with mod docs requirements, this should have an introduction. At least a sentence. Is that doable? (Applies to the other two high-level assemblies as well.)

Reply:
I moved some sentences from the "providers" modules to this intro. They make more sense here.